tools/codesigningtool/codesigningtool.py

```diff
@@ -120,16 +122,116 @@ def _certificate_fingerprint(identity):
   fingerprint = fingerprint.replace(":", "")
   return fingerprint
 
+def _certificate_common_name(cert):
+  _, subject, _ = execute.execute_and_filter_output([
+    "/usr/bin/openssl",
+    "x509",
+    "-noout",
+    "-inform",
+    "DER",
+    "-subject"
+  ], inputstr=cert, raise_on_failure=True)
+  subject = subject.strip().split('/')
+  cert_cn = [f for f in subject if "CN=" in f][0]
+  cert_cn = cert_cn.replace("CN=", "")
```

Comment:
I guess we should assume it's possible. I'll add the check instead of accessing `[0]` here and make it return `None`. Tested locally and is all good since at the call site the comparison is `if identity == common_name:`